### ISSUE TITLE
Is it possible to do live view and tethered capture at the same time?

### ISSUE THREAD

**Issue by 19hours** [gphoto/gphoto2#502]
Can I combine these two features together?

**Comment by msmeissn** [comment]
There is just 1 usb data/control flow possible.

You can trigger mix retrieving liveview and real captures from within 1 thread, but during the real capture no liveview frames will be captured of course.

**Comment by 19hours** [comment]
Yes during the capture no live view frames are needed, but how do I do both simultaneously? I'm currently using capture preview to do a live view by combining the frames. Can I just run the other command to listen to tethered captures?

gphoto2 --tethered-capture
